templates/includes/install-problems.hbs

```diff
@@ -11,7 +11,7 @@
       <li><h4>Verify that you used the correct image files</h4>Some devices like the Asus Zenwatch 2, the Fossil Gen4 models and the MTK6580 watches have more than one system image supplied, for different variants with slightly different hardware. Most watches have a graphical fastboot menu that shows the correct codename for the watch. Others need to be identified using the hardware differences noted above in the install section.</li>
       <li><h4>Clean your watch and charger contacts</h4>Even in case the fastboot flash process succeeds with no errors, it can still silently fail due to connection problems caused by dirty contacts. Clean the pins and pads with isopropyl alcohol. Ideally using that old toothbrush you keep around for electronics cleaning or a microfiber cloth. Carefully sanding the contacts (e.g. with a nailfile) can help in case of obvious corrosion.</li>
       <li><h4>Ensure a stable USB connection</h4>Put the watch and your computer on the same flat surface. Try to not move the cable connecting your computer and the watch during the whole flashing process. In case of worn or corroded contacts and pogo-pins that lost their springiness, it might be useful to attach the watch to its cradle or charging cable using duct tape or rubber bands.</li>
-      <li><h4>Repeat the flash process</h4>Some users have reported needing up to five flashing attempts for a successful result.<br>There is a small chance that the image files were corrupted during download. We do not currently provide checksums for the files. However, if you download the files again under a different filename and then compare them with <code>sha256sum filename</code>, the checksums must match.</li>
+      <li><h4>Repeat the flash process</h4>Some users have reported needing up to five flashing attempts for a successful result.<br>There is a small chance that the image files were corrupted during download. To guard against this, it's recommended to verify the checksums.  Download the <code>SHA512SUMS</code> checksum file from the same directory as the image files and verify them with <code>sha512sum --check SHA512SUMS</code>. The checksums for both files will be checked.</li>
```

Comment:
What would you think about having

```
"Download the  <a href="https://release.asteroidos.org/nightlies/{{.}}/SHA512SUMS">SHA512SUMS</a><code>SHA512SUMS</code></a> file and verify them with...
```

Here and not introducing a button for hashes at the top of the install page.

I personally find that checksums are only really worth checking in case of troubleshooting (btw have you *ever* downloaded a file with an incorrect checksum over http ?) and therefore it seems we can avoid one unnecessary step in the "critical user journey" of installing the OS and only keep the link at the time when it's actually relevant.

Note I removed "from the same directory as the image files" from your text as the user isn't really expected to know these files are in the same directory (they can just click the link)

Reply:
Done.  It's a bit more complex because some pages have multiple watches but this takes care of all that using templates.